### ISSUE TITLE
Conduct a Competitive Analysis

### ISSUE THREAD

**Issue by neslihanu** [FreeUKGen/FreeBMD2#58]
Collect and compare data about our competitors' websites; what they are offering and how is the service designed.

**Comment by neslihanu** [comment]
Competitive Analysis at the moment:
https://docs.google.com/spreadsheets/d/1TQuTuGixBCHLZuww8C7YYFutEKl7YeS0SHxgXOsU9gk/edit?usp=sharing

**Comment by DeniseColbert** [comment]
Thanks @neslihanu. Could you change permissions so that we can edit (to comment)?

**Comment by neslihanu** [comment]
Permissions changed.

https://docs.google.com/spreadsheets/d/1TQuTuGixBCHLZuww8C7YYFutEKl7YeS0SHxgXOsU9gk/edit?usp=sharing

**Comment by DeniseColbert** [comment]
Neslihan has completed the analysis as far as it would inform the search page (which is the priority right now) but there is still an analysis of competitors' search results pages to be done at the relevant time.

**Comment by neslihanu** [comment]
I`ve added the analysis of competitors` search results page and this is the final doc to review:

https://docs.google.com/spreadsheets/d/1TQuTuGixBCHLZuww8C7YYFutEKl7YeS0SHxgXOsU9gk/edit?usp=sharing

**Comment by PatReynolds** [comment]
Might be worth adding FamilySearch and UKBMD to competitor analysis.

**Comment by neslihanu** [comment]
Pat, I`m sorry.. the competitor analysis takes a lot of time. 4 months ago, before starting it, I talked to Denise and we decided the competitors which we should include in the analysis. At the moment, we are working on "Search Results" page and there are many things to do. I believe, this should be my priority. Would it be OK, if we keep the analysis as it is ? thanks

**Comment by PatReynolds** [comment]
That's fine - we can close this afternoon.

**Comment by richpomfret** [comment]
To add in FamilySearch and then UKBMD - @DeniseColbert to look at this.

**Comment by DeniseColbert** [comment]
FamilySearch complete

**Comment by richpomfret** [comment]
Closing - new story (241) to take this further. Thanks @DeniseColbert.
